### PR TITLE
71 view individual skill

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#71 
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/HqIndividualSkill/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    getData();
+    getActivities();
+  }, [id]);
+
+  return (
+    <S.Wrapper>
+      <BackLink />
+      <Row mt="5">
+        <Col w={[4, 8, 8]}>
```
Comment (thejoefriel) [inline client/src/pages/HqIndividualSkill/index.js:47]: Maybe add a bit of paddingRight here (e.g. 16px) for desk top just to give a bit more space between the title and edit button

File: client/src/pages/HqIndividualSkill/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      </Row>
+      <Row mt="5">
+        <Col w={[4, 12, 12]}>
+          <T.H4 mb="5">Activities</T.H4>
+        </Col>
+        {!activities.loading ? (
+          <>
+            {activities?.data?.length ? (
+              activities.data.map(({ title, id }) => (
+                <Col w={[4, 4, 4]}>
```
Comment (thejoefriel) [inline client/src/pages/HqIndividualSkill/index.js:71]: add margin bottom to these. you can see multiple ones are causing an issue when they wrap

image

File: client/src/pages/HqIndividualSkill/index.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    getActivities();
+  }, [id]);
+
+  return (
+    <S.Wrapper>
+      <BackLink />
+      <Row mt="5">
+        <Col w={[4, 8, 8]}>
+          <T.H3>{skill.loading ? <Skeleton active /> : skill.title}</T.H3>
+          <T.BodyB color="gray" mt="4" mb="4">
```
Comment (thejoefriel) [inline client/src/pages/HqIndividualSkill/index.js:49]: should be BodyR

File: client/src/pages/HqIndividualSkill/style.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  }
+`;
+
+export const ActivityCard = styled(Link)`
+  padding: 1rem;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.03), 0px 1px 8px rgba(0, 0, 0, 0.05),
```
Comment (thejoefriel) [inline client/src/pages/HqIndividualSkill/style.js:22]: get this shadow from the theme

**Review by thejoefriel — CHANGES_REQUESTED** [review]

File: client/src/pages/HqIndividualSkill/index.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/pages/HqIndividualSkill/index.js:47]: done

File: client/src/pages/HqIndividualSkill/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/HqIndividualSkill/index.js:71]: done, but not sure if this solves the issue, if you want we can make the height of the card fixed?

File: client/src/pages/HqIndividualSkill/index.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/pages/HqIndividualSkill/index.js:49]: done

File: client/src/pages/HqIndividualSkill/style.js (hunk 4 above)
Comment (RamyAlshurafa) [reply client/src/pages/HqIndividualSkill/style.js:22]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**